src/Psalm/Internal/Analyzer/ClassAnalyzer.php

```diff
@@ -397,17 +397,14 @@ public function analyze(
             }
         }
 
-        foreach ($storage->attributes as $i => $attribute) {
-            AttributeAnalyzer::analyze(
-                $this,
-                $class_context,
-                $attribute,
-                $class->attrGroups[$i],
-                $storage->suppressed_issues + $this->getSuppressedIssues(),
-                1,
-                $storage
-            );
-        }
+        AttributesAnalyzer::analyze(
+            $this,
+            $class_context,
+            $storage,
+            $class->attrGroups,
+            1,
```

Comment:
could we have AttributesAnalyzer::TARGET_DESCRIPTIONS public so that we can use class consts types instead of literals?